app/serializers/site_category_serializer.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
 class SiteCategorySerializer < BasicCategorySerializer
-  attributes :allowed_tags, :allowed_tag_groups, :allow_global_tags, :read_only_banner
+  attributes :allowed_tags,
+             :allowed_tag_groups,
+             :allow_global_tags,
+             :read_only_banner,
+             :form_template_ids
```

Comment:
This will still result in an N+1 when we're loading categories. We need to eagerload this here:

https://github.com/discourse/discourse/blob/52d4de7b45eb1b7c7997da2ba8bff83a3c22a3a5/app/models/site.rb#L75-L82